Files/UserControls/Widgets/DrivesWidget.xaml

```diff
@@ -12,8 +12,8 @@
     xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
     Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
     mc:Ignorable="d">
-    <StackPanel Spacing="12">
-        <TextBlock
+	<StackPanel Spacing="12">
```

Comment:
Don't change tabulation.

Reply:
Visual studio actually did it automatically. Should I close this and create a pr with changing it from a text editor?

Reply:
You can change VS configuration to use four spaces as tabulation. No need to open another PR.

Reply:
Okay. Will make the suggested changes.

Reply:
Made the changes to tabulation as suggested. Please let me know if this is alright.